internal/helper/helper.go

```diff
@@ -103,3 +106,68 @@ func GetCopyright(content string) string {
 
 	return ""
 }
+
+// GetCopyrightText ...
+func GetCopyrightText(path string) string {
+	r := reader.New(path)
+	c := r.StringFromFile()
+	ind := strings.Index(c, "Copyright (c)")
+	if ind < 0 {
+		return ""
+	}
+	copyWrite := strings.Split(c[ind:], "\n")
+	return strings.TrimSuffix(copyWrite[0], "\r")
+}
+
+// GetJSLicense ...
+func GetJSLicense(path string, pkName string, licenses map[string]string, modPath string, modManifest string) string {
```

Comment:
Use GetLicenses method instead

Reply:
> What happens if you dont find any dependencies for npm? also, we need to make sure our error messages are consistent with other modules. plz review go module

confirmed! the two errors are consistent